src/toml.rs

```diff
@@ -350,150 +432,31 @@ impl TomlParser {
             if self.cur == '\0' {
                 return Ok(TomlTok::Eof);
             }
-            match self.cur {
-                ',' => {
+
+            #[allow(unreachable_patterns)]
+            match self.cur as u32 {
+                // ,
+                0x2C => {
```

Comment:
I would really prefer if these stayed as `char`s instead of converting to `u32` first. It might make the codegen a tiny bit worse, but I really doubt it.

Reply:
Absolutely, will do.

Sorry but I'm a bit confused - by codegen are you referring to generating the match statement from the ABNF?